<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Authentication]">
  <test thread-count="5" name="LoginTest(failed)">
    <parameter name="correctUsername" value="test1@automationpractice.com"/>
    <parameter name="baseUrl" value="http://automationpractice.com/index.php"/>
    <parameter name="invalidUsername" value="automationpractice.com"/>
    <parameter name="correctPassword" value="28328719@Aut?!"/>
    <parameter name="invalidPassword" value="32408508e0ttu"/>
    <parameter name="inCorrectPassword" value="243544657@efg"/>
    <parameter name="invalidEmail" value="test212@automationpractice.com"/>
    <classes>
      <class name="autopractice.tests.LoginpageTest">
        <methods>
          <include name="beforeClass"/>
          <include name="loginWithValidPassword" invocation-numbers="0"/>
          <include name="beforeMethod"/>
          <include name="afterClass"/>
          <include name="afterTest"/>
          <include name="beforeSuite"/>
          <include name="afterMethod"/>
          <include name="afterSuite"/>
          <include name="beforeTest"/>
        </methods>
      </class> <!-- autopractice.tests.LoginpageTest -->
    </classes>
  </test> <!-- LoginTest(failed) -->
</suite> <!-- Failed suite [Authentication] -->
